osu.Game.Tests/Visual/Multiplayer/TestSceneMultiplayerMatchSubScreen.cs

```diff
@@ -183,6 +206,7 @@ public void TestFreeModSelectionHasAllowedMods()
         }
 
         [Test]
+        [FlakyTest]
```

Comment:
Yeah no. You've added this attribute to every test in this file? I can't be expected to not look into these tests after doing something like this.

Reply:
Go for it 👍 . I've spent hours on this and so have you in the past. I don't think it's worth my own time but it's causing the most failures of any tests.